.travis.yml

```diff
@@ -31,7 +36,7 @@ matrix:
     php: 7.2
     env: WP_VERSION=latest WP_MULTISITE=0 RUN_E2E=1
     addons:
-      chrome: stable
+      chrome: beta
```

Comment:
Why do we need to run Chrome beta? Is this the change that fixed the e2e tests? Asking just to so we have this change documented somewhere.

Reply:
Yes, there is a bug in webdriver shipped with chrome stable that is the causing this. I see now harm in using beta though, might help us pick up browser changes earlier as well.